steam_wrapper/steam_wrapper.py

```diff
@@ -243,10 +245,45 @@ def configure_shared_library_guard():
         if mode > 1:
             os.environ["LD_BIND_NOW"] = "1"
 
+def setup_compat_tool_extensions(current_info):
+    compat_tool_dest = Path('.local/share/Steam/compatibilitytools.d')
```

Comment:
I wonder, why this path is relative?

Reply:
Doesn't matter, the script initially chdirs to home to have consistent paths.